test/functional/tests/ci/test_basic.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from core.test_run import TestRun
+from api.cas.version import CasVersion
+from test_utils.filesystem.file import File
+
+
+def test_cas_version():
+    """
+    title: Check if CAS is installed
+    description:
+      Check if CAS is installed with --version flag and later
+      checks if components version is consistent with version file
+    pass criteria:
+     - casadm command succeeds
+     - Versions are matched from cmd and file in /var/lib/opencas/cas_version
+    """
+    cmd = f"casadm --version -o csv"
+    output = TestRun.executor.run(cmd)
+    if output.exit_code != 0:
+        TestRun.LOGGER.error(f"Command should return version")
```

Comment:
You can use `TestRun.executor.run_expect_success(cmd)`. It's more concise and does exactly the same thing

Reply:
Done :)